assets/src/edit-story/components/library/panes/text/textPresets.js

```diff
@@ -69,6 +69,7 @@ const DEFAULT_PRESET = {
   width: DEFAULT_WIDTH.DEFAULT,
   textAlign: 'center',
   borderRadius: DEFAULT_TEXT_BORDER_RADIUS,
+  fontFamily: 'Google Sans',
```

Comment:
Ditto here, we don't want default elements to use Google Sans. It's not an option in the font picker.